Tests/MergeL10nTests/ParserTests.swift

```diff
@@ -96,4 +96,116 @@ class ParserTests: XCTestCase {
 
         XCTAssertEqual("", rest)
     }
+
+    func testRequiresCommentShouldFailWhenThereIsNoComment() throws {
+        let file =
+        """
+        /* Comment for title */
+        "L10n.MyModule.SomeView.title" = "This is the title";
+        "L10n.MyModule.SomeView.body" = "This is the body";
+        /* Comment for caption */
+        "L10n.MyModule.SomeView.caption" = "This is the caption";
+        """
+
+        let expectedResult = [
+            LocalizedStringEntry(
+                key: "L10n.MyModule.SomeView.title",
+                value: "This is the title",
+                comment: "Comment for title")
+        ]
+
+        let (match, rest) = LocalizedStringFile.parser().run(file)
+
```

Comment:
Can you put „requiresComments: true“ manually here? I could not match test name + implementation at first. Would make it clearer, I think.

Reply:
Good point. One sec.

Reply:
done!